generators/generator-bot-adaptive/generators/app/index.js

```diff
@@ -107,10 +107,19 @@ module.exports = class extends BaseGenerator {
         ? `"${this.applicationSettingsDirectory}"`
         : defaultSettingsDirectory;
 
+    let settingsIncludePath = this.applicationSettingsDirectory;
```

Comment:
I'd use `path.join` for this, especially since path separators are platform specific.
```
> path.join('foo/', '/') === path.join('foo', '/')
true
```

This could be:
```js
const settingsIncludePath = path.join(this.applicationSettingsDirectory, path.sep);
```